README.md

```diff
@@ -1,3 +1,125 @@
+
+## Demo Setup
+
+First, you'll set up the demo.
+
+1. Change directory to `js`:
+
+```
+cd js
+```
+
+2. Update the salt nonce in `deploy-safe.ts` to `0x` followed by a few randomly chosen digits. For example,
+
+This salt nonce is used later to create the new Safe address.
+
+```ts
+const SALT_NONCE = "0x185593";
+```
+
+3. Run the `deploy-safe.ts` script to deploy the Safe smart account.
+
+```
+yarn deploy
+```
+
+> For the purposes of this demo, we're deploying to the Goerli test network. You can view your Safe on [Goerli Etherscan block explorer](https://goerli.etherscan.io/).
+
+4. Create a `.env` file at the root of the repo and add in the Safe address:
+
+You should have received the Safe address after deploying it.
+
+Copy and paste this address into your `.env` file:
+
+```bash
+SAFE_ADDRESS="0x..."
+```
+
+5. Finally, add the a Safe transaction signer by running:
+
+```
+yarn add-owner
+```
+
+Congratulations! You're done configuring the demo and deploying your own Safe smart account. 🥳
+
+### Demo Scripts
+
+The scripts below show complex transactions that your smart account can execute. Run `yarn` and then the script name to execute it.
+
+| Script              | Description                                                   |
+| ------------------- | ------------------------------------------------------------- |
+| `execute`           | Updates a stored value on a smart contract                    |
+| `execute-sponsored` | Uses the Gelato relay Wrap to execute a sponsored transaction |
+| `execute-multisend` | Sends transactions to multiple recipients                     |
+
+## Multi-platform Support: Rust
+
+One of the powerful advantages of the AA Wrap is that it can be used in any app, as long as the app has a Polywrap client library installed.
+
+The above scripts were run in a JavaScript environment. We've also prepared scripts that run in **Rust** below.
+
+First, you need to be in the `rs` directory.
+
+Then, update the salt nonce in the `deploy.rs` file:
+
+```rs
+    let deployment_input = DeploymentArgs {
+        input: DeploymentInput {
+            safe_account_config: AccountConfig {
+                owners: vec![signer_address.unwrap()],
+                threshold: 1,
+            },
+            safe_deployment_config: DeploymentConfig {
+                salt_nonce: "0x...".to_string(),
+            },
+            connection: None,
+        },
+    };
+```
+
+Then run:
+
+```
+cargo run --bin --release <script>
+```
+
+Where `<script>` is one of the following:
+
+| Script       | Description                     |
+| ------------ | ------------------------------- |
+| `deploy`     | Deploys your Safe smart account |
+| `add_owners` | Add owners to the Safe          |
+| `get_owners` | Get the list of owners          |
```

Comment:
we can also add sponsor transaction https://github.com/cbrzn/safe-playground/blob/main/rs/src/bin/sponsor_transaction.rs